cmd/api-linter/integration_test.go

```diff
@@ -184,7 +199,6 @@ func runLinter(t *testing.T, proto, config string) string {
 		"--cfg=" + configPath,
 		"--out=" + outPath,
 		"--proto_path=" + workdir,
```

Comment:
If we change the current directory to workdir, I don't think we need to point proto_path to it again.